### PR TITLE
Use FlaskGroup to create the CLI interface

### PR DESCRIPTION
**Pull request by alastair**
This will automatically create the app and wrap commands in a flask context, which will connect to any required services (e.g., database, cache)

### CODE REVIEW

**Comment by alastair** [comment]
@paramsingh ready

File: worker_manage.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import dataset_eval.evaluate
+import hl_extractor.hl_calc
+import webserver
+
+cli = FlaskGroup(add_default_commands=False, create_app=webserver.create_app)
+
+
+@cli.command('hl_extractor', help='High-level extractor tool')
+@click.option('-t', default=1, type=int)
```
Comment (paramsingh) [inline worker_manage.py]: Docstrings for this and the other command would be good to have.

Comment (paramsingh) [reply worker_manage.py]: Also, maybe using `threads` instead of `t` would be better.

Comment (alastair) [reply worker_manage.py]: I moved the help arguments to a docstring (click uses these as help if not explicitly set) and also cleaned up the docs in the main manage.py.

Added --threads as a long argument format.

Removed all create_app that wasn't needed

**Pull request merged**